### PR TITLE
cgmgr: reorder setting of sched_load_balance for pod cgroup

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

/kind bug
#### What this PR does / why we need it:
If we call mgr.Apply() first, libcontainer's cpusetCopyIfNeeded() will copy the parent cpuset and set load balancing to 1 by default. This causes the kernel to set the cpus to not load balanced for a brief moment which causes churn.

instead, create the cgroup and set load balance, then have Apply() copy the values into it.

#### Does this PR introduce a user-facing change?

```release-note
fix a bug in cpuset load balancing where cpusets flip between load balanced and not due to an ordering issue.
```

### CODE REVIEW

File: internal/config/cgmgr/cgmgr_linux.go [hunk 1]
```diff
@@ -180,23 +180,28 @@ func createSandboxCgroup(sbParent, containerCgroup string) error {
...
+	// Note: this should be done before Apply(-1) below, as Apply contains cpusetCopyIfNeeded(), which will
+	// populate the cpuset with the parent's cpuset. However, it will be initialized to sched_load_balance=1
+	// which will cause the kernel to move all cpusets out of their isolated sched_domain, causing unnecessary churn.
 	if !node.CgroupIsV2() {
 		path := mgr.Path("cpuset")
 		if path == "" {
 			return fmt.Errorf("failed to find cpuset for newly created cgroup")
 		}
-		return libctr.WriteFile(path, "cpuset.sched_load_balance", "0")
+		if err := os.Mkdir(path, 0o755); err != nil {
```
Comment (rphillips) [inline internal/config/cgmgr/cgmgr_linux.go:196]: We should probably catch the Exists case.

```
err != nil && !os.IsExist(err)
```

**Review by kolyshkin — APPROVED** [review]
lgtm

**Comment by rphillips** [comment]
lgtm

**Pull request merged**